ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/_repo-layout/create-run/-create-run-utils.ts

```diff
@@ -35,6 +35,16 @@ const keyValueSchema = z.object({
 
 const packageManagerOptionsSchema = z.object({
   enabled: z.boolean(),
+  mustRunAfter: z
+    .array(
+      z
+        .string()
+        // Only accept valid package manager IDs to the string array in the form
```

Comment:
As you know, I'm a fan of having dots at the end of sentences, even if they're just code comments 😉